<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MixerWidget</class>
 <widget class="QWidget" name="MixerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="btGetMute">
     <property name="text">
      <string>GetMute</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="cbGetMute">
     <property name="text">
      <string>muted</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btSetMute">
     <property name="text">
      <string>SetMute</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cbSetMute">
       <property name="text">
        <string>mute</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbSetMuteAck">
       <property name="text">
        <string>Ack</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btGetVolume">
     <property name="text">
      <string>GetVolume</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="sbGetVolume">
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btSetVolume">
     <property name="text">
      <string>SetVolume</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSpinBox" name="sbSetVolume">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbSetVolumeAck">
       <property name="text">
        <string>Ack</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
